librosa/effects.py

```diff
@@ -660,3 +661,68 @@ def preemphasis(y, coef=0.97, zi=None, return_zf=False):
         return y_out, z_f
 
     return y_out
+
+
+def deemphasis(y, coef=0.95, zi=None):
+    """De-emphasize an audio signal with the inverse operation of preemphasis():
+
+    If y = preemphasis(x, coef=coef, zi=zi), the deemphasis is:
+        x[i] -> y[i] + coef * x[i-1]
```

Comment:
again, i would prefer an equal sign rather than an arrow here
(`->` is not an assignment operator in Python like it is in R or Mathematica ; and besides, i would argue that the arrow should point leftward in this case)

Reply:
Fixed in https://github.com/librosa/librosa/pull/1293/commits/c25e8c220b0e565473c16eeed82babed4ff1b1bd. Sorry I missed this in the first round.